cosmicfishpie/configs/default_survey_specifications/Euclid-Photometric-ISTF-Pessimistic.yaml

```diff
@@ -1,15 +1,16 @@
 specifications:
-  'z_bins': [0.001, 0.418, 0.560, 0.678, 0.789, 0.900, 1.019, 1.155, 1.324, 1.576, 2.50]
+  'z_bins_ph': [0.001, 0.418, 0.560, 0.678, 0.789, 0.900, 1.019, 1.155, 1.324, 1.576, 2.50]
```

Comment:
I will add the other files in a patch for

Reply:
yeah, we need the other experiments. TODO for a new MR